style.css

```diff
@@ -0,0 +1,163 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url('images/bg-img.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+html {
+  scroll-behavior: smooth;
+}
```

Comment:
- [ ] The font-family is expected to be: Cocogoose, Lato. As seen in the given design please make the change.

Reply:
fixed